language.md

```diff
@@ -53,3 +53,4 @@ Maligayang Bati! [Philippines](Philippines/salamat.md)
 Εάν μιλάτε ελληνικά, συνεχίστε [εδώ](greek/marshmallow.md)
 
 Ja runā latviski, [dodies šeit](./latvian/zefirs.md).
+Ako govoris hrvatski, [klikni ovdje](croatian/sljez.md)

```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.